packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -64,24 +63,6 @@ class Paginator extends React.Component {
     };
   }
 
-  handleOnKeyDown(index) {
```

Comment:
How is the is logic being handled now? It doesn't seem like this state update got pulled forward unless I'm missing it

Reply:
The key down styles are being handled [directly in the button](https://github.com/cerner/terra-core/blob/2b3154ad8c49b58953711badfaa65d391fe0e718/packages/terra-paginator/src/_PaginatorButton.jsx#L62).

Reply:
I added onKeyDown for the selection to point to the page change function (since they were nearly identical) and added a check for key presses.